tests/console/yast2_apparmor.pm

```diff
@@ -33,8 +33,10 @@ sub run {
         send_key 'alt-e';
     }
     assert_screen 'yast2_apparmor_enabled';
+
     # part 1: open profile mode configuration and check toggle/show all profiles
-    send_key 'alt-n';
+    send_key (sle_version_at_least('15') ? 'alt-c' : 'alt-n');
```

Comment:
```perl
send_key 'sle_version_at_least('15') ? 'alt-c' : ' alt-n';
```

but I assume it is rather

```perl
send_key (!sle_version_at_least('15') && !leap_version_at_least('15.0')) ? 'alt-n' : 'alt-c';
```

Reply:
I have not yet tested it on Leap, but I would assume the same, will start Leap verification after implementing the suggestions

Reply:
I guess it's same for TW, as we have new gui there as well